extensions/google-translate/src/translate.tsx

```diff
@@ -31,34 +31,38 @@ export default function Command(): ReactElement {
 
     const promises = Promise.all([
       translate(toTranslate, {
-        from: preferences.lang1,
-        to: preferences.lang2,
+        from: undefined,
+        to: preferences.lang1,
       }),
       translate(toTranslate, {
-        from: preferences.lang2,
-        to: preferences.lang1,
+        from: undefined,
+        to: preferences.lang2,
       }),
```

Comment:
Now it looks like you can't really select FROM - TO language, because it will be "auto" all the time, I'm not sure that this is good

Let's add those "auto" as an option, so if anybody want to use auto, they will use it explicitly

> Also, I keep the 2nd language as it would helpful for learning a language.
Let's leave this extension only for translations, so users will be able to translate from any autodetected language to their language

Reply:
Also google translate is not very good platform to learn english, it would be better to create some another extension for learning purpose

And it will be good to use another API (not google translate), because if 429 happen in those extension, it will also affect this extension 😅

Reply:
So we only call API once as below, and adding "auto" to the first language option?
```
translate(toTranslate, {
        from: preferences.lang1,
        to: preferences.lang2,
})
```

Reply:
Yes, correct